tests/foreman/api/test_convert2rhel.py

```diff
@@ -355,9 +351,6 @@ def test_convert2rhel_centos_with_pre_conversion_template_check(
         synchronous=False,
         data={
             'job_template_id': template_id,
-            'inputs': {
```

Comment:
This change is part of an ongoing feature for Convert2Rhel improvement so I would suggest to attach jira link here as well with Verifies docstring.

Reply:
its just a removal of input, not addition, so why to keep Verifies inside docstrings, it doesn't make sense, right?

Reply:
This is functionality change in convert2rhel so I am in favor of keeping jira in docstring.

Reply:
yes, I understand, but we're not adding new change, its just a removal of existing option which was non-default